src/arns.lua

```diff
@@ -935,12 +919,8 @@ function arns.recordIsActive(record, timestamp)
 		return true
 	end
 
-	-- record starts before the current timestamp and ends after the current timestamp
-	return record.startTimestamp
-			and record.startTimestamp <= timestamp
-			and record.endTimestamp
-			and record.endTimestamp >= timestamp
-		or false
+	-- to avoid pruning on forked state for records that start int he future, only check the end timestamp against the current timestamp
```

Comment:
necessary as we will launch with start dates in the future, this would force those records to be pruned out